src/components/AppNavigation/Settings/SettingsAddressbookSharee.vue

```diff
@@ -126,3 +128,14 @@ export default {
 	},
 }
 </script>
+<style lang="scss" scoped>
+.addressbook-sharee__utils {
+	text-overflow: ellipsis;
+}
+
+.addressbook-sharee__utils label {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	width: 107px;
```

Comment:
```suggestion
	width: 107px !important;
```

This rule is overwritten otherwise.

Reply:
do we need the same for master, or only here?

Reply:
Only here